testchain/node/src/chain_spec.rs

```diff
@@ -340,5 +351,11 @@ fn testnet_genesis(
 			replace_griefing_collateral: FixedU128::checked_from_rational(1, 10).unwrap(), // 10%
 		},
 		nomination: NominationConfig { is_nomination_enabled: false },
+		dia_oracle_module: DiaOracleModuleConfig {
+			authorized_accounts: authorized_oracles,
+			supported_currencies: vec![AssetId::new(b"Bitcoin".to_vec(), b"BTC".to_vec())],
```

Comment:
I guess we should change this away from btc?

Reply:
Done.